cmake/FindOpenSSL.cmake

```diff
@@ -71,7 +71,7 @@ IF(WIN32)
   endif()
 ENDIF(WIN32)
 
-IF(ANDROID)
+IF(ANDROID AND WITH_FREERDP_OPENSSL)
```

Comment:
This seems broken as now in the android case no library detection is done, if not also `WITH_FREERDP_OPENSSL` is set.
Would suggest to do this differently by providing a default name `ANDROID_SSL_LIBRARY_NAME` that can be overridden from external sources.